fluent-dom/src/dom_localization.js

```diff
@@ -150,11 +160,17 @@ export default class DOMLocalization extends Localization {
    */
   disconnectRoot(root) {
     this.roots.delete(root);
-    // Pause and resume the mutation observer to stop observing `root`.
+    // Pause the mutation observer to stop observing `root`.
     this.pauseObserving();
-    this.resumeObserving();
 
-    return this.roots.size === 0;
+    if (this.roots.size === 0) {
+      this.mutationObserver = null;
+      this.windowElement = null;
```

Comment:
I'm not sure if this would be useful, but should we clear `this.pendingRAF` here as well, just to be safe?